extensions/azuremonitor/themes/kuskus-azuremonitorlogs-explorer-dark.json

```diff
@@ -0,0 +1,12 @@
+{
+	"tokenColors": "./Kuskus.AzureMonitorLogs.Dark.tmTheme",
+	"colors": {
+		"editor.background": "#222224",
+		"editorCursor.foreground": "#F8F8F0",
+		"editor.foreground": "#F8F8F2",
+		"editorWhitespace.foreground": "#3B3A32",
+		"editor.lineHighlightBackground": "#3D4247",
+		"editor.selectionBackground": "#1268AC"
+	},
+	"name": "[Kuskus] Azure Monitor Logs (Dark)"
```

Comment:
Shouldn't the names here and in the tmTheme files be different?

Reply:
Great catch. Thanks! I changed the names.